Buildings/Fluid/Humidifiers/EvaporativeCoolers/IndirectWet.mo

```diff
@@ -0,0 +1,259 @@
+within Buildings.Fluid.Humidifiers.EvaporativeCoolers;
+model IndirectWet "Indirect wet evaporative cooler"
+  extends Buildings.Fluid.Interfaces.PartialFourPortParallel(
+    redeclare final package Medium1=MediumPri,
+    redeclare final package Medium2=MediumSec);
+
+  replaceable package MediumPri = Modelica.Media.Interfaces.PartialMedium
+    "Medium 1 in the component"
+    annotation (choices(
+      choice(redeclare package Medium = Buildings.Media.Air "Moist air"),
+      choice(redeclare package Medium = Buildings.Media.Water "Water"),
+      choice(redeclare package Medium =
+        Buildings.Media.Antifreeze.PropyleneGlycolWater (
+        property_T=293.15,
+        X_a=0.40) "Propylene glycol water, 40% mass fraction")));
+
+  replaceable package MediumSec = Modelica.Media.Interfaces.PartialMedium
+    "Medium 2 in the component"
+    annotation (choices(
+      choice(redeclare package Medium = Buildings.Media.Air "Moist air")));
+
```

Comment:
I don't need to see the need to start a new naming pattern and use `MediumPri` and `MediumSec`. Better use good comments such as "Medium to be cooled" and "Medium for outside air to which heat is rejected". That would help the user more than `Pri` and `Sec`.